docs/container-set-template.md

```diff
@@ -116,3 +116,52 @@ Example B: Lopsided requests, e.g. `a -> b` where `a` is cheap and `b` is expens
 Can you see the problem here? `a` only has small requests, but the container set will use the  total of all requests. So it's as if you're using all that GPU for 10h. This will be expensive.
 
 Solution: do not use container set when you have lopsided requests.
+
+## `retryStrategy` usage
+
+> v3.3 and after
+
+You can set a `retryStrategy` to apply to all containers of a container set.
+
+This currently works differently from [retries](retries.md) for other template types.
+You can only set the `duration` between each retry and the total number of `retries`.
+
+See an example below:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  name: containerset-with-retrystrategy
+  annotations:
+    workflows.argoproj.io/description: |
+      This workflow template is used to create a workflow with containerset with retrystrategy.
+spec:
+  entrypoint: containerset-retrystrategy-example
+  templates:
+    - name: containerset-retrystrategy-example
+      containerSet:
+        retryStrategy:
+          retries: "2"
+        containers:
+          - name: success
+            image: python:alpine3.6
+            command:
+              - python
+              - -c
+            args:
+              - |
+                print("hi")
+          - name: fail-retry
+            image: alpine:latest
+            command: [ sh, -c ]
+            args: [ "echo intentional failure; exit 1" ]
+          - name: invalic-command
+            image: python:alpine3.6
+            command:
+              - invalid
+              - command
+```
+
+!!! NOTE
+    A container set will not be retried if a container's `command` cannot be located. As it will fail each time, the retry logic is short-circuited.
```

Comment:
```suggestion
!!! Note
    A container set will not be retried if a container's `command` cannot be located.
    As it will fail each time, the retry logic is short-circuited.
```

same as my [previous comment](https://github.com/argoproj/argo-workflows/pull/12809/files#r1538681610)
- Use title case "Note" for consistency with other areas of the docs
- Prefer 1 sentence per line of markdown, per [style guide](https://argo-workflows.readthedocs.io/en/release-3.5/doc-changes/)